CRM/Aoservicelisting/Form/ProviderApplicationForm.php

```diff
@@ -27,27 +21,22 @@ public function setDefaultValues() {
       $defaults['listing_type'] = 1;
     }
 
-    $loggedInContactId = $this->getLoggedInUserContactID();
-    if (!empty($loggedInContactId)) {
-      $relationship = civicrm_api3('Relationship', 'get', [
-        'contact_id_a' => $loggedInContactId,
-        'relationship_type_id' => 74,
-      ]);
-      if (!empty($relationship['count'])) {
-        $this->organizationId = $relationship['values'][$relationship['id']]['contact_id_b'];
-        $this->set('organizationId', $relationship['values'][$relationship['id']]['contact_id_b']);
+    if (!empty($this->_loggedInContactID)) {
+      if (!empty($this->organizationId)) {
+        // isn't it obvious that a person is only allowed to submit the application only after agreeing to the waiver it is a required field to proceed
+        $defaults['waiver_field'] = 1;
```

Comment:
@monishdeb My thinking about not prepopulating this is that i feel like there should be a 'reaffirming' of the waiver but may appreicate @JoeMurray 's thoughts on this. 

To summarise, When someone is logged in goes to update their/their org's service listing should the waiver checkbox be checked by default, I had it as No, Monish's chode will check it by default

Reply:
@seamuslee001 yeah I thought when a primary contact submits an application he/she has to check the waiver checkbox to proceed, as its required field. So when the loggedin user visits the edit page again. it should be checked as what is stored, isn't it?

Reply:
Note: The setting default is only on the edit page.

Reply:
@monishdeb from scrum channel

"SL: 1) when updating a listing should the waiver field be prechecked (given they would have had to check it initially)
JM: 1) No, make them positively agree."